pages/getting-started/sending-your-first-email.mdx

```diff
@@ -16,54 +16,185 @@ export default ({ children }) => <Layout meta={meta}>{children}</Layout>;
+
+### Schedule your newsletter email
+
+You don't have to send an email as soon as you write it. You can schedule it to send later, at a specific date or time. You can do this by clicking the “Scheduling” button at the bottom of your draft.
+
+To learn more, read [Scheduling an Email](/getting-started/scheduling-an-email).
+
+## Images
+
+Buttondown uses Amazon S3 (a common file hosting platform) to safely and securely store your images. Images you upload to Buttondown last indefinitely by default. This means that even if you were to delete the email in which images are referenced, the images themselves still exist on Buttondown's servers.
+
+<!-- TKTK What file formats are supported? Also should add real procedure here. -->
+
+There's no option in the Buttondown UI to delete uploaded images, but you can do so with the API.
+
+## Custom unsubscribe links
+
+Buttondown automatically appends an unsubscribe link at the end of all emails you send, but you may want to manually place one in your email as well (for instance, if you send your newsletter infrequently, you might place it at the top of the email). The unsubscribe link for your newsletter is unique. You can generate it with the `{{ unsubscribe_url }}` shortcode and use that in place of a URL in an email.
+
+Here is an example unsubscribe link in Markdown:
 
 ```handlebars
 [Click here to unsubscribe!]({{unsubscribe_url}}).
 ```
 
-(This link will render on draft emails, but the resultant URL won't do anything of importance. I'm working on a better solution so you can 'click through' and verify that the URL is well-formed.)
+This link renders on draft emails, but the URL won't do anything if you click it.
 
 ## Tags and segmentation
 
-Buttondown offers _tags_ to do this; subscribers can have one or more tags.
+Tags let you create subgroups of your subscribers. Subscribers can have one or more tags, so if you want to include someone in more than one tag group, you can.
 
-You can send emails including (or excluding!) certain tags, making it easy to
-tailor your newsletters' content to who's going to enjoy it the most or find it the most relevant
+You can send emails including certain tags, making it easy to tailor your newsletter's content to who's going to enjoy it the most or find it the most relevant.
 
-There are many ways to apply a tag to a subscriber, once you've created them:
+There are a few different ways to apply a tag to a subscriber:
 
+- Tagging subscribers manually
 - Customizing your [subscription form](https://blog.buttondown.email/2019/07/17/customizing-your-subscription-form-to-tag-new-subscribers)
-- Manually tagging them
-- Setting them via the [API](https://docs.buttondown.email/api-reference/introduction)
+- Setting tags with the [API](/api-reference/introduction)
+
+<!-- TKTK Should document how to manually tag subscribers, at least -->
```

Comment:
@jmduke I couldn't get this modal to appear anywhere in the emails page, or by searching for it. How do I find it?

Reply:
By going to a subscriber's detail page!

<img width="1313" alt="image" src="https://github.com/buttondown/docs/assets/461046/50c1b0f0-1db1-4f64-a72c-b1430393b65a">